### DISCUSSION TITLE
What mode does PRELU support?

### DISCUSSION THREAD

**Discussion by sandyhouse** [apache/incubator-mxnet#16677] [Q&A] [answered]
There are three modes for the PRELU activation, i.e., channel-wise mode (an alpha value per channel), element-wise (an alpha value per element) mode and share-channel mode (all elements share the same alpha value). I am not sure what mode does PRELU support. If it supports more than one mode, how to configure?

**Comment by ddavydenko** [comment]
@mxnet-label-bot add [Question]

**Comment by szha** [comment, marked as answer]
I think all modes are supported in the prelu in Gluon, and they are activated with broadcast semantics.
